turborepo-vercel/packages/ui/package.json

```diff
@@ -11,7 +11,8 @@
   ],
   "scripts": {
     "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
-    "dev": "tsup src/index.tsx --format esm,cjs --watch --dts --external react"
+    "dev": "tsup src/index.tsx --format esm,cjs --watch --dts --external react",
+    "typecheck": "tsc -b"
```

Comment:
this one really can't have the `-b` flag otherwise we emit files which we don't want as they're generated by tsup